test/unit/DCAHub/dca-hub-position-handler.spec.ts

```diff
@@ -688,10 +688,12 @@ contract('DCAPositionHandler', () => {
         const withdrawer = await readArgFromEventOrFail(response, 'WithdrewMany', 'withdrawer');
         const withdrawRecipient = await readArgFromEventOrFail(response, 'WithdrewMany', 'recipient');
         const positions = await readArgFromEventOrFail<any>(response, 'WithdrewMany', 'positions');
-        const withdrew = await readArgFromEventOrFail(response, 'WithdrewMany', 'withdrew');
+        const withdrew = await readArgFromEventOrFail<BigNumber[]>(response, 'WithdrewMany', 'withdrew');
         expect(withdrawer).to.equal(owner.address);
         expect(withdrawRecipient).to.equal(recipient);
-        expect(withdrew).to.eql([constants.ZERO, constants.ZERO]);
```

Comment:
Not sure why this doesn't work anymore. Probably has to do with the dependencies updates

Reply:
Hm... That sucks. Yeah, it's a dependencies update, for sure, not ours.